metacat-common/src/main/java/com/netflix/metacat/common/dto/BaseDto.java

```diff
@@ -35,7 +35,7 @@
  */
 //TODO: All DTO's should be READ-ONLY
 public abstract class BaseDto implements Serializable {
-    private static MetacatJsonLocator metacatJsonLocator = new MetacatJsonLocator();
+    protected static final MetacatJsonLocator METACAT_JSON_LOCATOR = new MetacatJsonLocator();
```

Comment:
Don't want to create instances in the deserializeObjectNode/ serializeObjectNode BaseDTO.